NuGet.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
GitHub won't let me comment on the signature.p7s files but I don't think those should be checked in unless the ref assemblies are reproducible - otherwise using these packages will fail on platforms that verify NuGet signatures.

Reply:
Removed.